src/pages/edge/adobe-journey-optimizer/push-notification/tabs/push-payload.md

```diff
@@ -0,0 +1,94 @@
+---
+noIndex: true
+---
+
+<Variant platform="android" api="payload-keys" repeat="3"/>
+
+* Adobe Journey Optimizer uses [data messages](https://firebase.google.com/docs/cloud-messaging/concept-options#data_messages) with custom key-value pairs to send push notifications to client apps.
+* When using [manual display and tracking](../android/manual-display-and-tracking.md), use the [MessagingPushPayload](../../public-classes-and-enum/messaging-push-payload.md) class to extract the payload values.
```

Comment:
Have you tested these links that they work? Links within tabs are... weird.

Reply:
![Screenshot 2023-10-27 at 9 49 31 AM](https://github.com/AdobeDocs/aep-mobile-sdkdocs/assets/8909148/d2f2d7c6-b30e-42c1-9819-948316de1630)

Reply:
they worked great testing on dev

Reply:
Got it - I'm always worried about clickthrough links on tabs, since they're really tricky to get right since they're not the exact ones that match with what you get on a normal markdown file, which is just why I wanted to have you double check them :P